cmake/modules/Findbabeltrace.cmake

```diff
@@ -16,7 +16,7 @@ find_program(BABELTRACE_EXECUTABLE
   NAMES babeltrace babeltrace-ctf)
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(BABELTRACE DEFAULT_MSG
+find_package_handle_standard_args(babeltrace DEFAULT_MSG
```

Comment:
so the all-upper-case versions are OK, but CryptoPP is not OK, because "the result-variable can be set either to either <UPPERCASED_NAME>_FOUND or <OriginalCase_Name>_FOUND" and "Other names for the result-variable are not allowed"? (In the case of CryptoPP, the "OriginalCase_Name" is "cryptopp", right?)

This would explain the bug report.